engine/src/main/scala/cromwell/server/CromwellShutdown.scala

```diff
@@ -0,0 +1,220 @@
+package cromwell.server
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import akka.Done
+import akka.actor.{ActorRef, ActorSystem, CoordinatedShutdown, _}
+import akka.http.scaladsl.Http
+import akka.pattern.{AskTimeoutException, GracefulStopSupport}
+import akka.routing.Broadcast
+import akka.stream.ActorMaterializer
+import akka.util.Timeout
+import com.typesafe.config.ConfigFactory
+import cromwell.engine.workflow.WorkflowManagerActor.{AbortAllWorkflowsCommand, InterruptNewWorkflowStart}
+import cromwell.services.SingletonServicesStore
+import cromwell.util.GracefulShutdownHelper.ShutdownCommand
+import org.slf4j.LoggerFactory
+
+import scala.concurrent.duration._
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success}
+
+/**
+  * Collection of methods and object used to control Cromwell graceful shutdown process.
```

Comment:
yes, I was suggesting to get rid of the first part of the sentence and start it with "Used".  🙂